apps/AEPSampleApp/android/app/src/main/java/com/aepsampleapp/MainApplication.java

```diff
@@ -10,130 +10,93 @@
-      return mNewArchitectureNativeHost;
-    } else {
-      return mReactNativeHost;
-    }
+    return mReactNativeHost;
   }
 
   @Override
   public void onCreate() {
     super.onCreate();
-    // If you opted-in for the New Architecture, we enable the TurboModule
-    // system
-    ReactFeatureFlags.useTurboModules = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
     SoLoader.init(this, /* native exopackage */ false);
+    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+      // If you opted-in for the New Architecture, we load the native entry
+      // point for this app.
+      DefaultNewArchitectureEntryPoint.load();
+    }
+    ReactNativeFlipper.initializeFlipper(
+        this, getReactNativeHost().getReactInstanceManager());
     MobileCore.setApplication(this);
     MobileCore.setLogLevel(LoggingMode.VERBOSE);
-    initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
-  }
-
-  /**
-   * Loads Flipper in React Native templates. Call this in the onCreate method with something like
-   * initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
-   *
-   * @param context
-   * @param reactInstanceManager
-   */
-  private static void
-  initializeFlipper(Context context,
-      ReactInstanceManager reactInstanceManager) {
-    if (BuildConfig.DEBUG) {
-      try {
-        /*
-        We use reflection here to pick up the class that initializes Flipper,
-        since Flipper library is not available in release mode
-        */
-        Class<?> aClass = Class.forName("com.aepsampleapp.ReactNativeFlipper");
-        aClass
-            .getMethod("initializeFlipper", Context.class,
-                ReactInstanceManager.class)
-            .invoke(null, context, reactInstanceManager);
-      } catch (ClassNotFoundException e) {
-        e.printStackTrace();
-      } catch (NoSuchMethodException e) {
-        e.printStackTrace();
-      } catch (IllegalAccessException e) {
-        e.printStackTrace();
-      } catch (InvocationTargetException e) {
-        e.printStackTrace();
-      }
-      MobileCore.configureWithAppID("YOUR-APP-ID");
-      List<Class<? extends Extension>> extensions = Arrays.asList(
-          Lifecycle.EXTENSION,
-          Signal.EXTENSION,
-          Edge.EXTENSION,
-          com.adobe.marketing.mobile.edge.identity.Identity.EXTENSION,
-          EdgeBridge.EXTENSION,
-          Messaging.EXTENSION,
-          UserProfile.EXTENSION,
-          Assurance.EXTENSION,
-          Places.EXTENSION,
-          Target.EXTENSION,
-          Optimize.EXTENSION,
-          Consent.EXTENSION);
-      MobileCore.registerExtensions(extensions, o -> {
-        MobileCore.lifecycleStart(null);
-      });
-    }
+    MobileCore.configureWithAppID("YOUR-APP-ID");
+    List<Class<? extends Extension>> extensions = Arrays.asList(
+        Lifecycle.EXTENSION, Signal.EXTENSION, Edge.EXTENSION,
+        com.adobe.marketing.mobile.edge.identity.Identity.EXTENSION,
```

Comment:
We are missing direct identity registration.
Please add 
com.adobe.marketing.mobile.Identity.EXTENSION